cab/admin.py

```diff
@@ -13,6 +13,21 @@ class SnippetAdmin(admin.ModelAdmin):
     date_hierarchy = 'pub_date'
     search_fields = ('author__username', 'title', 'description', 'code',)
     raw_id_fields = ('author',)
+    actions = ['mark_as_inappropiate', 'mark_as_spam']
+
+    def mark_as_inappropiate(self, request, queryset):
+        for obj in queryset:
+            if not obj.flags.all():
```

Comment:
Should we consider the case where a user/admin has marked a snippet as inappropriate and it later gets marked as spam or vice versa? If so, this is not supported.

Reply:
Hi sorry I have taken so long to get back to you , I completely missed this.Thanks for the PR. Will go through it this week 👍

Reply:
Yes I think the intent is that multiple users can mark it as spam so we wont need the check